src/Options/FixtureOptions.php

```diff
@@ -300,14 +300,13 @@ public function getProjectTemplate(): string {
       return $this->options['project-template'];
     }
 
+    // @todo Point to latest version of DRP once 2.x for D10 stable fixtures
```

Comment:
As a rule, we shouldn't use abbreviations unless they're widely known. We have no idea who may be reading this code in the future or whether "DRP" will mean anything to them. "D10", for example, is common enough that we can use it.